lego/apps/users/notifications.py

```diff
@@ -30,3 +36,47 @@ def generate_push(self):
             context={"weight": penalty.weight, "event": penalty.source_event.title},
             instance=penalty,
         )
+
+
+class InactiveNotification(Notification):
+
+    name = INACTIVE_WARNING
+
+    def generate_mail(self):
+        max_inactive_days = self.kwargs["max_inactive_days"]
+
+        return self._delay_mail(
+            to_email=self.user.email,
+            context={
+                "name": self.user.first_name,
+                "username": self.user.username,
+                "last_login": str(self.user.last_login.date()),
```

Comment:
Is this human-readable or just DD.MM.YYYY? Some date formatting would be nice, f.ex DD month YYYY. But not very important...

Reply:
This is in the format YYYY-MM-DD, quite human-readable